eng/Versions.props

```
@@ -11,7 +11,7 @@
     <VersionFeature>00</VersionFeature>
     <VersionPrefix>$(VersionMajor).$(VersionMinor).$(VersionSDKMinor)$(VersionFeature)</VersionPrefix>
     <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
-    <PreReleaseVersionIteration>7</PreReleaseVersionIteration>
+    <PreReleaseVersionIteration>8</PreReleaseVersionIteration>
```

According to WillG, this is how we mark the release for rc1.
```suggestion
    <PreReleaseVersionIteration>1</PreReleaseVersionIteration>
    <PreReleaseBrandingLabel>rc</PreReleaseBrandingLabel>
```